openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -679,7 +680,9 @@ def run(self, *, dry=False, verbose=True,
     def _execute(
         self, ctx: gufe.Context, **kwargs,
     ) -> dict[str, Any]:
-        outputs = self.run(scratch_basepath=ctx.scratch, shared_basepath=ctx.shared)
+        with without_oechem_backend():
```

Comment:
Actually we might need to review this, I suspect we'll run into trouble when we start offering folks the ability to choose what charge derivation method they can have (for example if they want elf charges)